tiatoolbox/tools/patchextraction.py

```diff
@@ -298,23 +241,13 @@ def __init__(
             (self.patch_size[1] - 1) / 2
         )
 
-    def merge_patches(self, patches=None):
-        """Merge patch is not supported by :obj:`PointsPatchExtractor`.
-        Calling this function for :obj:`PointsPatchExtractor` will raise an error. This
-        overrides the merge_patches function in the base class :obj:`PatchExtractor`
-
-        """
-        raise MethodNotSupported(
-            message="Merge patches not supported for PointsPatchExtractor"
-        )
-
 
 def get_patch_extractor(method_name, **kwargs):
     """Return a patch extractor object as requested.
 
     Args:
-        method_name (str): name of patch extraction method, must be one of "point",
-          "fixedwindow", "variablewindow".
+        method_name (str): name of patch extraction method, must be one of "point" or
+          "sliding".
```

Comment:
need to change to "slidingwindow".

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/121/commits/0f9566e304bcd67e8a62aea686c04e8fb1a1cc59